### PR TITLE
Add instructions to global env to setup cloudscale metrics collector

### PR DESCRIPTION
**Pull request by ccremer**
This PR adds instructions on how to save the cloudscale metrics collector, required for billing AppCat service "ObjectStorage".

See also:
* https://hub.syn.tools/cloudscale-metrics-collector/how-tos/installation.html
* https://github.com/vshn/cloudscale-metrics-collector/

## Checklist

- [x] Try to isolate changes into separate PRs (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `change`, `decision`, `requirement/quality`, `requirement/functional`, `dependency`
      as they show up in the changelog
- [x] Link this PR to related issues if applicable.

### CODE REVIEW

File: docs/modules/ROOT/pages/how-to/vshn-example/activate-global.adoc [hunk 1]
```diff
@@ -46,3 +48,16 @@ Client Roles = appuio-cloud-realm
 Assigned Roles = view-users
 ----
 <1> The tabs are visible after saving the new user first.
+
+== Configure Cloudscale Metrics Collector
+
+. Login to Vault.
+. Login to control.cloudscale.ch.
+. Create a new API token within the `VSHN AppCat objects-lpg-2` project, give it a memorable name like `cloudscale-metrics-collector` and read-only access.
```
Comment (bastjan) [inline docs/modules/ROOT/pages/how-to/vshn-example/activate-global.adoc:56]: Is this value hard-coded somewhere? `VSHN AppCat objects-lpg-2`

**Review by bastjan — APPROVED** [review]

File: docs/modules/ROOT/pages/how-to/vshn-example/activate-global.adoc (hunk 1 above)
Comment (ccremer) [reply docs/modules/ROOT/pages/how-to/vshn-example/activate-global.adoc:56]: WDYM with hardcoded? It's a changeable display name, manually set up by someone who has enough permissions on control.cloudscale.ch to create projects.
I wrote the instructions as if you're ever to recreate the global environment, that's the current project to use. When we launch the objectstorage on other clusters, we need to update these instructions probably

Comment (bastjan) [reply docs/modules/ROOT/pages/how-to/vshn-example/activate-global.adoc:56]: As some parts are very specific we could maybe also mention the specific vault url + login url?

Comment (ccremer) [reply docs/modules/ROOT/pages/how-to/vshn-example/activate-global.adoc:64]: added a snippet

**Pull request merged**